source/elements/oneMKL/source/domains/rng/device_routines/device-rng-bits.rst

```diff
@@ -0,0 +1,61 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_bits:
+
+bits
+====
+
+Generates bits of underlying engine (BRNG) integer recurrence.
+
+.. rubric:: Description
+
+The ``bits`` class object is used in ``generate`` and ``generate_single`` functions to provide integer 
+random values. Each integer can be treated as a vector of several bits. In a truly random generator, these 
```

Comment:
we don't have true random number generator in device api, so we can remove the first part of the sentence